buildSrc/src/main/kotlin/tanvd/grazi/Dependencies.kt

```diff
@@ -20,10 +20,23 @@ fun ExternalModuleDependency.ltExcludes() {
     exclude("javax.xml.bind", "jaxb-api")
     exclude("org.glassfish.jaxb", "jaxb-runtime")
     exclude("com.fasterxml.jackson.core", "jackson-databind")
+    exclude("com.esotericsoftware.kryo", "kryo")
 
-    // used only in languagetool-core test
+    exclude("org.apache.commons", "commons-lang3")
+//    exclude("org.apache.commons", "commons-csv")
```

Comment:
So, what's with commented parts?

Reply:
They are not in IDEA, but somehow it works without them...

Reply:
Gonna inspect it in the next pull request, where we removing LangDetector and html